src/Integration.Vsix.UnitTests/SonarLintTagger/IssueMarkerTests.cs

```diff
@@ -25,20 +25,22 @@
 using SonarLint.VisualStudio.Core.Analysis;
 using SonarLint.VisualStudio.Core.Suppression;
 using SonarLint.VisualStudio.Integration.Vsix;
+using SonarLint.VisualStudio.IssueVisualization.Models;
```

Comment:
I fixed up all of the existing tests so they pass, but in fact quite a few of the affected tests in the older text buffer/tagger classes will be deleted later.

Perhaps I should have deleted all of the old tagger behaviour and tests first to avoid having to fix them up. However, it's been useful to be able run the code and manually validate that I haven't broken anything.